### DISCUSSION TITLE
uninstall openmobu

### DISCUSSION THREAD

**Discussion by im-pro-viz** [Neill3d/OpenMoBu#5] [Q&A]
Hi, Segey! Thank you for great plug-ins and scripts. Is there a way to uninstall openmobu plugins and scrips, and return motionbuilder to its default settings. I found moplugs in Apps&Features (windows) list, but i can't find the openmobu,  though i installed it the sam way as the moplug, with .exe file. Добрый день, Сергей! Спасибо Вам за отличные инструменты. Скажите пожалуйста, есть ли способ удалить openmobu, что бы вернуть motionbuilder в дефолтное состояние? Molplugs есть в списке установленных программ в windows. А openmobu нет, хоть его так же устанавливал через .exe файл.

**Comment by Neill3d** [comment]
Добрый вечер, вот только что попробовал 2019_1_0b, удаляется через "установка и удаление программ". Установщик в moplugs используется абсолютно такой же. Возможно по какой-то причине установка не была завершена, попробуйте повторно установить.

**Reply by im-pro-viz** [reply]
Нет! переустановка не добавила в список openmobu. 
Решил задачу следующим образом. Может будет полезно.
Попробовав несколько комбинаций с установкой и удалением, выявил, что в зависимости от того, какой из 2-х (MoPlugs или OpenMoBu) устанавливается в первую очередь, в списке установленных программ windows, появляется только данный "плагин". При чем, если в списке, например только openMoBu, то при его удалении, предлагает удалить MoPlugs. И тоже самое наоборот. Но удаляются в обеих случаях оба. И MoPlugs, и OpenMoBu. Во всяком случае в motionbuilder исчезают все инструменты.

**Reply by Neill3d** [reply]
интересно, спасибо, что поделились

**Reply by Neill3d** [reply]
я получал обратную связь, что установщики не очень то и удобны, особенно если нужно поставить не весь набор, а только один или два плагина. Так что есть куда улучшать...
